manifests/vanilla/create-snapshot-validation-webhook.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2021 The Kubernetes Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  cat <<EOF
+Usage: Patch csi-snapshot-validatingwebhook.yaml with CA_BUNDLE retrieved from Kubernetes API server
+and create ValidatingWebhookConfiguration and snapshot-validation-service service using patched yaml file
+
+EOF
+  exit 1
+fi
+
+CA_BUNDLE=$(kubectl get configmap -n kube-system extension-apiserver-authentication -o=jsonpath='{.data.client-ca-file}' | base64 | tr -d '\n')
+
+namespace="kube-system"
+
+kubectl delete service snapshot-validation-service --namespace "${namespace}" 2>/dev/null || true
```

Comment:
Can you add comments before these "kubectl delete" to explain why these need to be deleted?

Reply:
Due to https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/968 change, this change will only post the yaml file. I will modify https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/954 to deploy the validation webhook. This way users need to run just a single script to deploy all the snapshot components.